### PR TITLE
ApiTest and Display

### PR DESCRIPTION
**Pull request by sl7v3**
Successful lol api access and display in LeagueStory/LeagueSummary section of profile page.
Will not be merged, just for anyone who needs reference to check it out. I will delete it, especially because of my API key in a few days, so grab it and change the API key to continue using it.

Helpful documentation to use API: http://www.rubydoc.info/gems/ruby-lol/0.11.6/Lol/

### CODE REVIEW

File: app/models/summoner.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+    require 'lol'
+    class Summoner < ActiveRecord::Base
+        
+        @@Summoner = Lol::SummonerRequest.new "833becde-d428-4d74-bd27-8418e1c0a33d", "na"
```
Comment (danfstucky) [inline app/models/summoner.rb]: Does this need to be a class variable?

**Comment by danfstucky** [comment]
I can't actually get this to run.  Does it work for you?

**Comment by danfstucky** [comment]
Also, the last line in a function is implicitly returned in ruby so the common practice is to only use the `return` keyword when explicitly necessary.  
You are also looking up the same information many times your summoner model.  You search lol's db with `@@Summoner.by_name(summoner_name)` 5 separate times.  It would be more efficient to search it once, store that data in an object and then simply retrieve what you need from it in each function.

**Comment by danfstucky** [comment]
We are using this [Ruby wrapper](https://github.com/mikamai/ruby-lol) as well.  It makes the code a bit cleaner and we already added the gem so might as well make use of it.  

Also, I don't think you are using the `:summoner` attr_accessor

File: config/secrets.yml [hunk 2]
```diff
@@ -10,8 +10,12 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+
+
+
 development:
   secret_key_base: 875c55a921da093b3c7d7c5be7ff363d6baa9091a0cd84f08e2c6908164d144b5a3a85c534416282ff691fdbf8382a4a5a8ede89d2a08f782f0cdd1de27c30e8
+  sulai_api_key: 833becde-d428-4d74-bd27-8418e1c0a33d
```
Comment (danfstucky) [inline config/secrets.yml]: Should this be in quotes or does it matter?

File: Gemfile.lock [hunk 3]
```diff
@@ -127,6 +139,7 @@ GEM
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
+    sqlite3 (1.3.11)
```
Comment (danfstucky) [inline Gemfile.lock]: You have duplicates of sqlite3 and also bcrypt.  Was there a reason for it or was that a mistake?

**Comment by sl7v3** [comment]
I figured it would be better to just merge the branch I created to fix the SSL error to this one instead of creating a new pull request/branch. I have copied the comments from that new branch/pull request here and yes this branch works on Windows now.

API test now works on Windows for me. Try pulling this branch and see if it works. The information I followed to solve the problem was buried deep in http://stackoverflow.com/questions/5720484/how-to-solve-certificate-verify-failed-on-windows
It talked about installing certified and requiring it in the model


This branch has not been merged. You and Dan need to check that it works and doesn't break anything before it can be merged to master. Thanks,Sulaimon Lasisi Top 5 Clifton Strengths - Consistency - Responsibility - Discipline - Futuristic - Analytical

    On Thursday, June 9, 2016 10:03 PM, jwiebe17 <notifications@github.com> wrote:


 Did you already merge the branch into master?—

**Comment by jwiebe17** [comment]
I can hopefully take a look today. Do I need to use rails console to test api or should it work straight up.

**Comment by sl7v3** [comment]
It should work without rails console
On Jun 13, 2016 10:10 AM, jwiebe17 <notifications@github.com> wrote:I can hopefully take a look today. Do I need to use rails console to test api or should it work straight up.

**Comment by danfstucky** [comment]
Nice job!  I pulled it and confirmed it works on my machine as well.  
+1
Were you planning on merging this or was it just a test?

**Comment by sl7v3** [comment]
Thanks Dan. It was just a test and a reference point for accessing the API if needed. I don't see what use merging it will do. However, we need to discuss what information to consume/display from the API going forward. 


Sent from my T-Mobile 4G LTE Device

<div>-------- Original message --------</div><div>From: Dan Stucky <notifications@github.com> </div><div>Date:06/13/2016  2:57 PM  (GMT-05:00) </div><div>To: danfstucky/LeagueSpec <LeagueSpec@noreply.github.com> </div><div>Cc: sl7v3 <seun.lasisi@yahoo.com>, Mention <mention@noreply.github.com> </div><div>Subject: Re: [danfstucky/LeagueSpec] ApiTest and Display (#18) </div><div>
</div>Nice job! I pulled it and confirmed it works on my machine as well.

+1
Were you planning on merging this or was it just a test?

**Pull request closed without merging**